src/jsm.c

```diff
@@ -4019,8 +4019,9 @@ js_cloneConsumerConfig(jsConsumerConfig *org, jsConsumerConfig **clone)
         for (int i = 0; (s == NATS_OK) && (i < org->FilterSubjectsLen); i++)
         {
             IF_OK_DUP_STRING(s, c->FilterSubjects[i], org->FilterSubjects[i]);
+            if (s == NATS_OK)
+                ++c->FilterSubjectsLen;
```

Comment:
nit: Equivalent here, but I prefer `c->FilterSubjectsLen++;` (this would match more rest of NATS C client code).